internal/cmd/rsl/log/log.go

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package log
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/gittuf/gittuf/internal/repository"
+	"github.com/spf13/cobra"
+)
+
+type options struct{}
+
+func (o *options) AddFlags(_ *cobra.Command) {}
+
+func (o *options) Run(_ *cobra.Command, _ []string) error {
+	repo, err := repository.LoadRepository()
+	if err != nil {
+		return err
+	}
+	entries, annotationMap, err := repository.LogRSLEntries(repo)
+	if err != nil {
+		return err
+	}
+	log := ""
+
+	for index := len(entries) - 1; index >= 0; index-- {
+		entry := entries[index]
+
+		log += fmt.Sprintf("entry %v", entry.ID)
+
+		skipped := false
+
+		if annotations, ok := annotationMap[entry.ID]; ok {
+			for _, annotation := range annotations {
+				if annotation.Skip {
+					skipped = true
+					break
+				}
+			}
+		}
+
+		if skipped {
+			log += " (Skipped Entry)\n"
+		} else {
+			log += "\n"
+		}
+
+		if annotations, ok := annotationMap[entry.ID]; ok {
+			for _, annotation := range annotations {
+				log += fmt.Sprintf("    Message:\n        %s\n", annotation.Message)
+			}
+		}
+
+		log += "\n"
+	}
+
+	cmd := exec.Command("echo", log)
```

Comment:
I'm still leaning towards creating some kind of `internal/display` package to handle these more involved interactions with stdout.

Reply:
Should this be incorporated in this PR, or a later one?

Reply:
I'd probably do it here for this one. Separately, a future PR could move logic from cmd for list-rules.

Reply:
For this, are you saying that the logic should be in internal/display, but the calling should still be in cmd, correct?

Reply:
yes

Reply:
Fixed